core-services/release-controller/_releases/release-origin-4.4.json

```diff
@@ -11,6 +11,16 @@
   "verify":{
     "aws":{
       "prowJob":{"name":"release-openshift-origin-installer-e2e-aws-4.4"}
+    },
+    "upgrade":{
+      "upgrade":true,
+      "optional":true,
+      "prowJob":{"name":"release-openshift-origin-installer-e2e-aws-upgrade"}
+    },
+    "upgrade-minor":{
```

Comment:
Remove this, no previous minor to upgrade from.

Otherwise looks good.

Reply:
done